### PR TITLE
Remove notices

### PR DESCRIPTION
**Pull request by brianteeman**
There really is no point in displaying a popup alert to say that there are alerts - thats all this page is for

This PR removes the popup that displays there is an error or that there is no error AND when there is no error then it is displayed as a message in the body instead.

### Before
chrome_2018-08-16_17-44-18
chrome_2018-08-16_17-48-42

### After
chrome_2018-08-16_17-59-39
chrome_2018-08-16_18-03-00

### CODE REVIEW

**Comment by infograf768** [comment]
I guess the string `COM_INSTALLER_MSG_WARNINGS_NOTICE` has to be deleted.

**Comment by brianteeman** [comment]
Good spot. I wrongly assumed it was a generic message used elsewhere. Removed it now

File: administrator/components/com_installer/tmpl/warnings/default.php [hunk 1]
```diff
@@ -33,6 +33,11 @@
 							<h4 class="alert-heading"><?php echo Text::_('COM_INSTALLER_MSG_WARNINGFURTHERINFO'); ?></h4>
 							<p class="m-b-0"><?php echo Text::_('COM_INSTALLER_MSG_WARNINGFURTHERINFODESC'); ?></p>
 						</joomla-alert>
+					<?php else: ?>
+						<joomla-alert type="info">
```
Comment (wilsonge) [inline administrator/components/com_installer/tmpl/warnings/default.php:37]: Does this come up with the aria alert like it did on the permissions fields?

Comment (brianteeman) [reply administrator/components/com_installer/tmpl/warnings/default.php:37]: good spot thanks but its correct

**Comment by brianteeman** [comment]
thanks

**Pull request merged**